README.md

```diff
@@ -68,7 +68,7 @@ Follow these steps to quickly install the binaries for `foundry-zksync`:
    Begin by cloning the `foundry-zksync` repository from GitHub. This will download the latest version of the source code to your local machine.
 
    ```bash
-   git clone git@github.com:matter-labs/foundry-zksync.git
+   https://github.com/matter-labs/foundry-zksync.git
```

Comment:
In general not an issue, just curious about the motivation to use https origin

Reply:
> In general not an issue, just curious about the motivation to use https origin

That's what it says in the main documentation https://docs.zksync.io/build/
https://github.com/matter-labs/zksync-web-era-docs

I just fixed a part of the code in line with it.

Reply:
@alexanderblv hope you are aware that this is not your PR.

Reply:
> @alexanderblv hope you are aware that this is not your PR.

I can't create a PR, but I created the issue that the part of PR was based on.